gtkmm-plplot/plotdatasurface.h

```diff
@@ -18,7 +18,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #ifndef GTKMMPLPLOT_PLOT_DATA_SURFACE_H
 #define GTKMMPLPLOT_PLOT_DATA_SURFACE_H
 
-#include <gtkmm-plplotconfig.h>
+#include <gtkmm-plplot/gtkmm-plplotconfig.h>
```

Comment:
This would break any software building against gtkmm-plplot since `gtkmm-plplotconfig.h` ends up after `make install` as `$prefix/lib/gtkmm-plplot-2.0/include/gtkmm-plplotconfig.h`.

Reply:
OK I understand.
The problem is that the file is not in the same location before and after installation.
One solution might be to put the file at the root of the project. So we could keep the include without the directory.

Reply:
This file is actually generated from `gtkmm-plplotconfig.h.in` when the `configure` script is run, and will be put into your build directory afterwards. If you want to use a cmake buildsystem instead you would still need to generate this file whenever cmake is run, as it contents depend on whether boost is present on as well as which version of PLplot is installed.